src/index.js

```diff
@@ -0,0 +1,15 @@
+import './main.scss';
+import  { sendRequest, createLi, URL }  from './modules/reExport'
+
+sendRequest('GET', URL, "todos")
+    .then(data => {
+        const elements = data.slice(0,10)
+        elements.forEach(
+          item => destructuringAndCreate(item)
```

Comment:
Вынос логики в destructuringAndCreate будет лишней в этом случае. destructuringAndCreate просто делигирует создание через createLi.

```suggestion
          item => {
                  const {completed, id, title, userId} = item
                  createLi(title, id, completed, userId)
          }
```